pyproject.toml

```diff
@@ -5,6 +5,46 @@ build-backend = "setuptools.build_meta"
 [project]
 dynamic = ["version"]
 name = "openlibrary-client"
+authors = [{name = "Internet Archive", email = "mek@archive.org"}]
+license = {text = "LICENSE"}
+description = "A Python client for Open Library"
+readme = "README.md"
+classifiers = [
+    "Development Status :: 2 - Pre-Alpha",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: GNU Affero General Public License v3",
```

Comment:
Probably not a blocker for this PR either, but it reminds me I have questioned this before: shouldn't this client simply be GPL v3?  

The Open Library codebase is AGPL because it is a server, ol-client isn't.